CMakeLists.txt

```diff
@@ -371,19 +408,26 @@ endif()
 
 target_compile_definitions(ufs_model PRIVATE "${_ufs_model_defs_private}")
 
-if(DATM)
+if(DATM OR CDEPS)
   target_link_libraries(ufs_model PUBLIC ufs w3nco::w3nco_d)
 endif()
 target_link_libraries(ufs_model PRIVATE ufs
                                         esmf
                                         NetCDF::NetCDF_Fortran
                                         ${MKL_LIB})
 
+if(S2S OR CMEPS OR CDEPS)
```

Comment:
Is this only true for PNETCDF used with S2S, CMEPS, or CDEPS or do you mean for these links to be included if PNETCDF is true?

Reply:
PNETCDF is only required when CMEPS and CDEPS is used. Plus, the io type need to be set to pnetcdf in the nems.configure. These changes will not be used in the near future since it requires new PIO installation with pnetcdf support and the FindPnetCDF.cmake to find the pnetcdf. This is currently in a branch of CMakeModules.